Twig/Extension/PostExtension.php

```diff
@@ -86,7 +87,15 @@ public function getPermalink($postId)
 
         $permalinkStructure = $this->optionExtension->getOption('permalink_structure', '')->getValue();
 
-        return $this->replacePostArguments($permalinkStructure, $post);
+        $relativeUrl = $this->replacePostArguments($permalinkStructure, $post);
+
+        if($isAbsolute) {
```

Comment:
@ruslan-polutsygan Can you please add a space after `if` statement? This is to be complient with http://www.php-fig.org/psr/psr-2/#1-1-example

Thank you